### DISCUSSION TITLE
High load / cpu usage since a few days for script/websocket-server.rb

### DISCUSSION THREAD

**Discussion by blu-IT** [modoboa/modoboa#2694] [Help] [answered]
Hello,

since a few day `htop` shows me a very high load of `script/websocket-server.rb -b 0.0.0.0 -p 6042 start` that was startet by modoboa.

When I reboot the server, for a few minutes load is OK, but then the process is started again by modoboa and shows a permanent high load.
I am not really sure, that this process really belongs to modoboa, because I also have Zammad with docker running on this host and Zammad processes are also shown as if they where started or belong to user modoboa, what is somekind of strange.

Any idea?

Thanks!

Regards

![htop](https://user-images.githubusercontent.com/13665806/202144715-420c3545-1e12-411c-bdcd-03070c04b503.png)

**Comment by tonioo** [comment, marked as answer]
@blu-IT This script has nothing to do with modoboa... And according to the script extension, it's a ruby script.

**Comment by blu-IT** [comment]
@tonioo Oh, OK. Then it is a zammad issue!

Thanks for yor quick feedback!
